release/models/network-instance/openconfig-evpn-types.yang

```diff
@@ -287,4 +294,10 @@ module openconfig-evpn-types {
       "Type of MAC address learning procedure";
   }
 
-}
+  typedef multicast-address {
```

Comment:
Is this address used in multiple places, since this typedef seems like it adds little value - given that it aliases the `ip-address` type?

Reply:
Ok, I removed the typedef and just use `type oc-inet:ip-address;` directly.

Also, has there been any consideration to add a `type oc-inet:multicast-address` under `openconfig-inet-types.yang` with a more restrictive pattern that only covers the multicast range?

Reply:
Changed to use ip-address type directly.